app/models/work.rb

```diff
@@ -56,6 +56,11 @@ class Work < ApplicationRecord
     end
   end
 
+  sig { returns(T.nilable(String)) }
+  def name
```

Comment:
I think this is a confusing method name. name/title are easily confused.  How about `title_or_default`?

Reply:
Yeah, I was originally going to use "display_title" but wanted to be consistent with something similar being done for collections here: https://github.com/sul-dlss/happy-heron/blob/main/app/components/dashboard/collection_component.rb#L16-L19

We may want to consider renaming that method to be consistent if we change it for works here.

Reply:
`display_title` would be okay too. If you want to be consistent, then putting it in the component (rather than the model) would be a good idea.  Then you could call it `title` as a component method wouldn't be expected to return exactly what a model method does.

Reply:
I just removed it from the model and added it as "title" to the `Works::DetailComponent`, which I then used on the Dashboard.  This also automatically fixes one more spot that I missed on the Work show page.